wiki/Tournaments/SOFT/3/en.md

```diff
@@ -33,19 +33,19 @@ tags:
 
 | Placing | Prize(s) |
 | :-: | :-- |
-| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | Profile badge and 9 months of osu!supporter |
-| ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | 6 months of osu!supporter |
-| ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | 4 months of osu!supporter |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, 9 months of osu!supporter tag |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 6 months of osu!supporter tag |
```

Comment:
I understand that "tag" has been added for consistency, but just "osu!supporter" reads way better

Reply:
i will fix this in a seperate PR i guess since this is a case for nearly all tournaments